content/zh/docs/tasks/observability/distributed-tracing/overview/index.md

```diff
@@ -1,27 +1,25 @@
 ---
-title: Overview
-description: Overview of distributed tracing in Istio.
+title: 概述
+description: Istio 中调用跟踪的概述。
 weight: 1
 keywords: [telemetry,tracing]
 aliases:
  - /zh/docs/tasks/telemetry/distributed-tracing/overview/
 ---
 
-Distributed tracing enables users to track a request through mesh that is distributed across multiple services.
-This allows a deeper understanding about request latency, serialization and parallelism via visualization.
+分布式跟踪可以让用户对跨多个分布式服务网格的1个请求进行跟踪分析。这样进而可以通过可视化的方式更加深入地了解请求的延迟，序列化和并行度。
 
-Istio leverages [Envoy's distributed tracing](https://www.envoyproxy.io/docs/envoy/v1.10.0/intro/arch_overview/tracing) feature
-to provide tracing integration out of the box. Specifically, Istio provides options to install various tracing backend
-and configure proxies to send trace spans to them automatically.
-See [Zipkin](../zipkin/), [Jaeger](../jaeger/) and [LightStep](/zh/docs/tasks/observability/distributed-tracing/lightstep/) task docs about how Istio works with those tracing systems.
+Istio 利用 [Envoy 的分布式跟踪](https://www.envoyproxy.io/docs/envoy/v1.10.0/intro/arch_overview/tracing) 功能提供了开箱即用的跟踪集成。确切地说，Istio 提供了安装各种各种跟踪后端服务的选项，并且通过配置代理来自动发送跟踪 spans 到跟踪后端服务。
 
-## Trace context propagation
+请参阅 [Zipkin](../zipkin/)， [Jaeger](../jaeger/) 和 [LightStep](/zh/docs/tasks/observability/distributed-tracing/lightstep/) 的任务文档来了解 Istio 如何与这些分布式跟踪系统一起工作。
 
-Although Istio proxies are able to automatically send spans, they need some hints to tie together the entire trace.
-Applications need to propagate the appropriate HTTP headers so that when the proxies send span information,
-the spans can be correlated correctly into a single trace.
+## 跟踪上下文传递{#trace-context-propagation}
 
-To do this, an application needs to collect and propagate the following headers from the incoming request to any outgoing requests:
+尽管 Istio 代理能够自动发送 spans，但是他们需要一些附加线索才能将整个跟踪链路关联到一起。
```

Comment:
spans -> span

Reply:
This word I hesitated，spans or span? 
yes It should be span in Chinese. And there is other I change back to span.

Reply:
fixed